pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -276,9 +276,6 @@ func (rq *ResourceQuotaController) syncResourceQuota(resourceQuota api.ResourceQ
 	dirty = dirty || (resourceQuota.Status.Hard == nil || resourceQuota.Status.Used == nil)
 
 	used := api.ResourceList{}
-	if resourceQuota.Status.Used != nil {
```

Comment:
there may be multiple controllers enforcing specific usage values.  for example, a quota may have a constraint for "mycustomresource/foo" that this controller can't count, but another can.  as a result, we need to preserve the used values for those resources, we don't understand.